rpcs3/Emu/RSX/Overlays/overlay_animation.cpp

```diff
@@ -1,19 +1,21 @@
-#include "stdafx.h"
+﻿#include "stdafx.h"
 #include "overlay_animation.h"
 #include "overlay_controls.h"
 
+#include <numeric>
+
 namespace rsx
 {
     namespace overlays
     {
         void animation_translate::apply(compiled_resource& resource)
         {
-            if (progress == vector3i(0, 0, 0))
+            if (current == vector3i(0, 0, 0))
```

Comment:
This animation idle condition no longer makes sense as the algorithm no longer uses motion vectors. It may be removed or a check for the active state used instead.